release notes/v0.46.0.md

```diff
@@ -0,0 +1,131 @@
+      key: open("key0.pem"),
+    }, // password omitted to demonstrate 'optional password'
+  },
+  "grpcbin.test.fakek6.io:9001": {
+    plaintext: false,
+    tls: {
+      cacerts: open("cacerts1.pem"),
+      cert: open("cert1.pem"),
+      key: open("key1.pem"),
+      password: "cert1-passphrase",
+    }, // cacerts as a 'string' to demonstrate string|string[] typing of cacerts
+  },
+};
+const clients = {
+  "grpcbin.test.notk6.io:9001": new grpc.Client(),
+  "grpcbin.test.fakek6.io:9001": new grpc.Client(),
+};
+...
+
+// k6 - VU code phase (3)
+if (__ITER === 0) {
+  clients["grpcbin.test.notk6.io:9001"]
+    .connect("grpcbin.test.notk6.io:9001", params["grpcbin.test.notk6.io:9001"]);
+  clients["grpcbin.test.notk6.io:9001"]
+    .connect("grpcbin.test.fakek6.io:9001", params["grpcbin.test.fakek6.io:9001"]);
+}
+...
+```
+</details>
+
+Thanks @chrismoran-mica for contribution 🙇‍♂️.
+
+### Cloud Output v2 [#3117](https://github.com/grafana/k6/issues/3117)
+
+After years of great service, we decided to refresh the k6 Cloud output introducing a more efficient end-to-end solution for ingesting the generated tests' metrics. The main change regards the protocol used for flushing metrics that is now a binary based payload over HTTP.
+
+The new output reduces the load generators' used resources for tests that produce many metrics. There is no significant difference in the user experience; it's expected to be the same. 
+
+The one thing worth highlighting is that the new output is strict about tags and drops tags if they are reserved:
+- `test_run_id` as it is reserved for internal k6 Cloud operations
+- any tag with a key that starts with two underscores (`__`), that is marked by Prometheus convention as reserved
+
+At the moment this is not yet the default Cloud output for the test runs executed from local machines (`k6 run -o cloud`), but it is expected to be transparently enabled in the upcoming weeks.
+
+The full list of related PRs: [#3104](https://github.com/grafana/k6/pull/3104), [#3108](https://github.com/grafana/k6/pull/3108), [#3120](https://github.com/grafana/k6/pull/3120), [#3125](https://github.com/grafana/k6/pull/3125), [#3162](https://github.com/grafana/k6/pull/3162), [#3169](https://github.com/grafana/k6/pull/3169), [#3182](https://github.com/grafana/k6/pull/3182), [#3186](https://github.com/grafana/k6/pull/3186), [#3187](https://github.com/grafana/k6/pull/3187), [#3193](https://github.com/grafana/k6/pull/3193), [#3195](https://github.com/grafana/k6/pull/3195), [#3206](https://github.com/grafana/k6/pull/3206), [#3226](https://github.com/grafana/k6/pull/3226), [#3157](https://github.com/grafana/k6/pull/3157), [#3172](https://github.com/grafana/k6/pull/3172).
+
+### `<big_feature_n>` `#pr`
+
+_what, why, and what this means for the user_
+
+### UX improvements and enhancements
+
+_Format as `<number> <present_verb> <object>. <credit>`_:
+- _`#999` Gives terminal output prettier printing. Thanks to `@person` for the help!_
+
```

Comment:
The version rendered of this section contains unexpected space between items. I don't know what is the specific reason, maybe it is this line.

```suggestion
```